index.rst

```diff
@@ -121,8 +128,9 @@ files take precedence.
 Supported Properties
```

Comment:
```suggestion
Supported Keys
```

Reply:
I think this should be "Supported Declarations" because this section of the document does cover supported keys, but also which values are supported for those keys; thus, the entire declaration support.

Reply:
Sounds good